flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/Catalog.java

```diff
@@ -311,6 +311,15 @@ void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ignoreIfE
     void alterTable(ObjectPath tablePath, CatalogBaseTable newTable, boolean ignoreIfNotExists)
             throws TableNotExistException, CatalogException;
 
+    /**
+     * If true, tables which do not specify a connector will be translated to managed tables.
+     *
+     * @see CatalogBaseTable.TableKind#MANAGED
+     */
+    default boolean supportsManagedTable() {
```

Comment:
I might miss a point in the original design, but why the catalog needs to support managed tables? Isn't the support already provided by CatalogManager itself?

Reply:
There are some catalogs that already use table without connector as their catalog internal table.

Reply:
You can see the first few replies of the FLIP discussion.